src/client/components/split-tile/split-tile.tsx

```diff
@@ -207,7 +208,7 @@ export class SplitTile extends React.Component<SplitTileProps, SplitTileState> {
 
   dragOver = (e: React.DragEvent<HTMLElement>) => {
     if (!this.canDrop()) return;
-    e.dataTransfer.dropEffect = "move";
+    if (!isInternetExplorer()) e.dataTransfer.dropEffect = "move";
```

Comment:
We prevent drop based on DragManager, we can leave dropEffect out for all browsers.